### DISCUSSION TITLE
EAB with Custom ACME Server

### DISCUSSION THREAD

**Discussion by los93sol** [nginx-proxy/acme-companion#927] [Ideas]
It would be nice if ACME_EAB_KID and ACME_EAB_HMAC_KEY would apply regardless if using ZeroSSL.  This is especially useful for custom ACME servers.

**Comment by buchdag** [comment]
Hi, can you give `nginxproxy/acme-companion:eab` a try and tell me if it work for you ?

It has `ACME_EAB_KID` / `ACME_EAB_HMAC_KEY` enabled no matter the `ACME_CA_URI` being used (image based on the [`global-eab`](https://github.com/nginx-proxy/acme-companion/tree/global-eab) branch).

**Reply by ElasMiratoris** [reply]
This branch works great for Sectigo!
Do you intend to merge this into main?

**Reply by buchdag** [reply]
@ElasMiratoris I still have to validate that it does not break compatibility with Zero SSL and I'll merge it into main.

**Comment by buchdag** [comment]
Done in #981

**Comment by los93sol** [comment]
Apologies for not responding, this is excellent, thank you!

**Reply by buchdag** [reply]
No worries ^^
